### PR TITLE
Epicenter 1201

### PR DESCRIPTION
**Pull request by mmrj**
minor docs updates

### CODE REVIEW

File: documentation/index.html.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ Learn more about advanced topics:
 
 1. Add the Flow.js required libraries to your project. Flow.js requires the following files:
 	* [`jquery.js`](http://jquery.com): document manipulation, DOM element selection, and event handling used by Flow.js
+		* NOTE: Flow.js requires version 2.x of `jquery.js`.
 	* [`lodash.js`](http://lodash.com): utilities and performance enhancements used by Flow.js; also used in [templating](#templates)
+		* NOTE: Flow.js requires version 2.x of `lodash.js`.
```
Comment (narenranjit) [inline documentation/index.html.md]: is this repeated twice or am i reading it wrong?

Comment (mmrj) [inline documentation/index.html.md]: one is lodash, one is query.

Comment (mmrj) [inline documentation/index.html.md]: let me know if there a more specific version of either we should be recommending.

Comment (narenranjit) [inline documentation/index.html.md]: sorry, i should stop commenting on Friday evenings :-/ Technically 2.x isn't a requirement for jQuery, just a recommendation - older 1.x versions should work fine, but okay to leave it as-is.

Comment (mmrj) [inline documentation/index.html.md]: sounds good. makes sense to me to leave 2.x because that's what a ui builder sim gives you. thanks!

**Pull request merged**